salt/reboot/init.sls

```diff
@@ -0,0 +1,42 @@
+##################################################
+# Configuration for the reboot manager
+##################################################
+
+# `max_holders` contains the maximum number of lock holders for the cluster.
+{% set max_holders = salt['pillar.get']('etcd:masters', '1')|int %}
+{% if max_holders > 1 %}
+  {% set max_holders = max_holders - 1 %}
```

Comment:
Max holders actually can be a completely different number based on `etcd:masters`.

- `etcd:masters == 1` (shouldn't happen) -> `max_holders: 1`
- `etcd:masters == 3` -> `max_holders: 1`
- `etcd:masters == 5` -> `max_holders: 2`
- `etcd:masters == 7` -> `max_holders: 3`
- `etcd:masters == 9` -> `max_holders: 4`

This is based on https://coreos.com/etcd/docs/latest/v2/admin_guide.html (Fault Tolerance Table).